ProjectManager/ProjectManager/View/AddView/ProjectAdditionScrollView.swift

```diff
@@ -118,7 +121,27 @@ final class ProjectAdditionScrollView: UIScrollView {
             scheduleDescriptionTextView.leadingAnchor.constraint(equalTo: shadowView.leadingAnchor),
             scheduleDescriptionTextView.topAnchor.constraint(equalTo: shadowView.topAnchor),
             scheduleDescriptionTextView.trailingAnchor.constraint(equalTo: shadowView.trailingAnchor),
-            scheduleDescriptionTextView.bottomAnchor.constraint(equalTo: shadowView.bottomAnchor)
+            scheduleDescriptionTextView.bottomAnchor.constraint(equalTo: shadowView.bottomAnchor),
+            scheduleDescriptionTextView.heightAnchor.constraint(
+                greaterThanOrEqualTo: self.heightAnchor,
+                multiplier: 0.48
+            )
         ])
     }
+    
+    func setContent(data: ProjectUnit) {
+        scheduleTitleTextField.text = data.title
+        scheduleDescriptionTextView.text = data.body
+        datePicker?.date = data.deadLine
+    }
+
+    private func configureInset() {
+        let contentInset = UIEdgeInsets(
+            top: 0.0,
+            left: 0.0,
+            bottom: 15,
+            right: 0.0)
```

Comment:
보통 줄바꿈 컨벤션을 이렇게 정의하면 아래와 같이 적용하는 경우가 많아요.

```suggestion
        let contentInset = UIEdgeInsets(
            top: 0.0,
            left: 0.0,
            bottom: 15,
            right: 0.0
        )
```

그리고, UI 관련 상수 값은 namespace에 정의해두고 사용하시는 편이 이후에 추적하기 편하실거에요. 저는 `Metric`이라는 이름으로 정의해서 사용하곤 해요.